mne/datasets/utils.py

```diff
@@ -98,12 +98,12 @@ def _get_path(path, key, name):
     # 3. get_config('MNE_DATA')
     path = get_config(key or 'MNE_DATA', get_config('MNE_DATA'))
     if path is not None:
-        if not op.exists(path):
+        if not op.exists(op.expanduser(path)):
```

Comment:
why not use pathlib throughout?
```suggestion
        path = Path(path).expanduser()
        if not path.exists():
```

and then the corresponding change below